src/app/clusters/thermostat-server/thermostat-server.h

```diff
@@ -105,10 +97,18 @@ class ThermostatAttrAccess : public chip::app::AttributeAccessInterface
     bool InAtomicWrite(CommandHandler * commandObj, EndpointId endpoint);
 
 private:
-    CHIP_ERROR AppendPendingPreset(Delegate * delegate, const Structs::PresetStruct::Type & preset);
+    CHIP_ERROR AppendPendingPreset(Thermostat::Delegate * delegate, const Structs::PresetStruct::Type & preset);
+
+    void OnFabricRemoved(const FabricTable & fabricTable, FabricIndex fabricIndex) override;
+
+    struct AtomicWriteState
+    {
+        bool inProgress;
+        ScopedNodeId nodeId;
+        EndpointId endpointId;
```

Comment:
```suggestion
        bool inProgress = false;
        ScopedNodeId nodeId;
        EndpointId endpointId = kInvalidEndpointId;
```

or does this generate too much extra code? Generally I would try to initialize everything to known defaults.

Reply:
Added initializers.